jina/parsers/peapods/pod.py

```diff
@@ -145,3 +145,13 @@ def mixin_k8s_pod_parser(parser):
         if _SHOW_ALL_ARGS
         else argparse.SUPPRESS,
     )
+
+    parser.add_argument(
+        '--k8s-envs',
```

Comment:
Why do we need a new argument? Cant we just use the [existing one](https://github.com/jina-ai/jina/blob/master/jina/parsers/peapods/pea.py#L48)?

Reply:
yes. @bwanglzu This variable `env` exists for `Pea`, `Pod` & `Flow`. Let's reuse them here.

Reply:
good, I'll make use `env`, thanks!